pages.de/linux/numlockx.md

```diff
@@ -0,0 +1,20 @@
+# numlockx
+
+> Steuern des Zahlensperrenstatus in X11-Sitzungen.
```

Comment:
I think most users would be confused by `Zahlensperrstatus`. The English page talks about `number lock` which isn't correct because it's called [`Num Lock`](https://en.wikipedia.org/wiki/Num_Lock). In German it's called [`Num-Taste`](https://de.wikipedia.org/wiki/Num-Taste) or `Num-Lock-Taste`.
```suggestion
> Steuere den Num-Lock-Tasten-Status in X11-Sitzungen.
```